### PR TITLE
Add ceph to apt-repos

### PR DESCRIPTION
**Pull request by msambol**

### CODE REVIEW

File: roles/apt-repos/defaults/main.yml [hunk 1]
```diff
@@ -33,3 +33,6 @@ apt_repos:
   erlang:
     repo: 'http://packages.erlang-solutions.com/debian'
     key_url: 'https://packages.erlang-solutions.com/debian/erlang_solutions.asc'
+  ceph:
+    repo: 'http://ceph.com/debian-'
```
Comment (zsais) [inline roles/apt-repos/defaults/main.yml:37]: The repo url resolves to a 404. Should the hyphen be removed?

Comment (msambol) [inline roles/apt-repos/defaults/main.yml:37]: We add `hammer` (the ceph version we want) to it in the `ceph-common` role.

Comment (zsais) [inline roles/apt-repos/defaults/main.yml:37]: Got it, thanks man

Comment (msambol) [inline roles/apt-repos/defaults/main.yml:37]: thank you, sir

**Pull request merged**